pupil_src/shared_modules/marker_tracker_3d/controller.py

```diff
@@ -0,0 +1,26 @@
+from marker_tracker_3d import optimization
+from marker_tracker_3d.basic_models import CameraModel, MarkerModel
+from marker_tracker_3d.camera_localizer import CameraLocalizer
+from marker_tracker_3d.marker_detector import MarkerDetector
+
+
+class Controller:
+    def __init__(self, storage, update_menu):
+        self.storage = storage
+        self.storage.camera_model = CameraModel(cameraMatrix=None, distCoeffs=None)
+        self.storage.marker_model = MarkerModel()
```

Comment:
The `marker_model` seems to be never used?

Reply:
`self.storage.marker_model` is passed to and used in `Optimization` and `Localization`